openpype/hosts/aftereffects/plugins/load/load_file.py

```diff
@@ -46,6 +46,10 @@ def load(self, context, name=None, namespace=None, data=None):
 
         frame = None
         if '.psd' in path:
+
+            comp_name = get_unique_layer_name(
```

Comment:
Why not simply adding the is_psd to the existing `comp_name = get_unique_layer_name(` line 28/29

Reply:
Because the modification with the Path object made in [404](https://github.com/quadproduction/issues/issues/404) is not yet deployed, so I can't verify yet with the "path.suffix == '.psd'" if it's a psd or not before entering the if condition.

Reply:
and in the one line 28, the is_psd is not necessary since by default, the is_psd is set to False in get_unique_layer_name